hack/components/git-utils.sh

```diff
@@ -18,6 +18,17 @@ function git-utils::fetch_component() {
     )
 }
 
+function git-utils::fetch_component_master() {
```

Comment:
This is not used

Reply:
I will remove it since we need a specific commit and not the latest master